app/javascript/packages/document-capture/components/address-search.spec.tsx

```diff
@@ -55,3 +57,40 @@ describe('AddressSearch', () => {
     await expect(handleLocationsFound).to.eventually.be.called();
   });
 });
+
+describe('AddressSearch analytics', () => {
+  // wanted to reflect an empty response
+  // let server: SetupServerApi;
+  // before(() => {
+  //   server = setupServer(
+  //     rest.post(LOCATIONS_URL, (_req, res, ctx) => res(ctx.json([{ name: 'Baltimore' }]))),
+  //     rest.post(ADDRESS_SEARCH_URL, (_req, res, ctx) => res(ctx.json([]))),
+  //   );
+  //   server.listen();
+  // });
+
+  // after(() => {
+  //   server.close();
+  // });
+
+  it('logs an event when clicking the call to action button', async () => {
```

Comment:
analytics test i'd like eyes on

Reply:
it seemed to me it wasn't truly working bc i changed the value of the error to something random and it still worked. Can confirm with your suggestion the test still passes.